src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/repositories/AuthorizationRepository.kt

```diff
@@ -24,13 +24,15 @@ interface AuthorizationRepository
     fun ensureGroupDoesNotHaveMember(userGroup: String, email: String)
     fun ensureUserGroupHasPermission(userGroup: String, permission: ReifiedPermission)
     fun ensureUserGroupDoesNotHavePermission(userGroup: String, permission: ReifiedPermission)
-    fun getPermissionsForUser(email: String): PermissionSet
+    fun getPermissionsForUser(email: String): List<String>
```

Comment:
For whatever reason the latest pac4j was unable to deserialise permissions in the session store correctly into a custom type, so have switched to using the built in `permissions` property in the session store which is a set of strings, and then doing the parsing after they are retrieved: https://github.com/vimc/orderly-web/pull/470/files#diff-3373578afc2188f7b7b007730bfb9454212eea38dd55de044a48bb0df8a1551fR17